docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -176,3 +176,32 @@ For the remaining users:
 
 .. literalinclude:: resource-groups-example.json
     :language: json
+
+.. _scheduleweight-ex:
+
+``schedulingWeight`` example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For each resource subgroup, the weighted scheduling is essentially a raffle
+ticket. If your ``schedulingWeight`` is 10, you’ve bought 10 tickets, and the
+other subgroup whose weight is 1 only bought 1. All of the tickets get thrown
+into the same jar, and Trino picks one ticket at random every time it has
+capacity to run a query. 
```

Comment:
I feel this is a non-inclusive example. Raffles are not a thing everywhere and we are writing for a global audience. Suggest:

Schedule weighting is a method of assigning a priority to a resource.  Higher priority resources are assigned a higher scheduling weight in order to ensure that they are run in a more timely manner.